ds-prep-python-functions/main.py

```diff
@@ -0,0 +1,43 @@
+bool_to_int = lambda value: 1 if value else 0
+print('bool_to_int(True) ', bool_to_int(True))
+print('bool_to_int(False) ', bool_to_int(False))
+
+get_smaller = lambda a, b: a if a < b else b
+print('get_smaller(16, 31) ', get_smaller(16, 31))
+print('get_smaller(253, 223) ', get_smaller(253, 223))
```

Comment:
Please redefine these without writing `if`-`else` expressions (built-in functions are ok):

---

![image](https://user-images.githubusercontent.com/16143771/148328239-d555821a-0af1-4182-8600-49507fda8b97.png)

Reply:
removed if else logic from lambda function